packages/babel-plugin-transform-remove-unused-catch-binding/src/index.js

```diff
@@ -0,0 +1,28 @@
+import syntaxOptionalCatchBinding from "babel-plugin-syntax-optional-catch-binding";
+
+export default function() {
+  return {
+    inherits: syntaxOptionalCatchBinding,
+
+    visitor: {
+      CatchClause(path) {
+        const binding = path.scope.getOwnBinding(path.node.param.name);
```

Comment:
The `binding` can be null. 

If the `actual.js` is 

```js
try {} catch {}
```

There is no param, and it'll throw when using `binding.something`. Can you also add a test case where the catch param is not present.